impl/src/main/java/io/kroxylicious/testing/kafka/common/Utils.java

```diff
@@ -205,6 +211,9 @@ private static CompletionStage<Void> createTopic(Integer expectedBrokerCount, Ad
                 .thenRun(() -> log.debug("Create future for topic {} completed.", CONSISTENCY_TEST))
                 .exceptionallyComposeAsync((throwable) -> {
                     log.warn("Failed to create topic: {} due to {}", CONSISTENCY_TEST, throwable.getMessage());
+                    if (throwable instanceof CompletionException && throwable.getCause() != null) {
+                        throwable = throwable.getCause();
```

Comment:
nit: I'm not a fan of re-assigning the parameter. I would prefer to see us delegate the handling of the throwable to a new method and call it with `throwable.getCause()` in this case and just `throwable` in others.

Reply:
Ok, I see how an extract method looks.  I'll include it as part of #138 as that changes the method too.